### ISSUE TITLE
Anomaly in presence of data gaps

### ISSUE THREAD

**Issue by paoladaulerio** [ESA-VirES/Aeolus-Client#43]
Selecting a time interval  to visualize  a product type  (e.g. L1B) which includes  some data gaps,  we found an anomaly (see the attached screenshot) both in the representation of ground track and in the analytic 2D plot. To be noticed that the problem even for a same product type is not always replicated, as in some cases data are presented as expected (see second attachment)

**Comment by santilland** [comment]
Thank you for the information. We detect gaps in the data by checking for a longer "jump" in the position of the measurement from one to the next. If there is a gap we check usually if it is an anti meridian crossing from longitude -180 to 180, but aeolus uses for some reason longitudes from 0 to 360, which is not usually how longitude is handled (see for example http://spatialreference.org/ref/epsg/wgs-84/).
In any case if the crossing is from 0 to 360 we connect the values to make them continuous.
In the case you are showing the data gap is exactly at that crossing, that is why it is actually connected. I will check if i can introduce an additional check to see if latitude changes larger then expected at the longitude crossing from 0 to 360, that should in theory solve the problem.

**Comment by santilland** [comment]
I added a new check for this case and deployed it to the client. This solves the issue (at least the one specified here). I will close the ticket, if you find a similar related issue for a combination of gap that has not been foreseen for some reason please feel free to create a comment here.

**Comment by paoladaulerio** [comment]
Found a similar issue (at least in terms of behaviour)  for the L1B data gap occurred on 28th Oct (approximatively in the time interval 21- 22)

**Comment by santilland** [comment]
After looking into the issue the problem is the end point of the previous curtain and the start point of the next curtain are very close together at the pole and the "jump" is below the normal threshold of what a gap is considered, so it is not separated correctly. I added a check also for gaps in time which should now also take this possibility into consideration.
I deployed the solution to the staging environment.
